packages/fether-react/src/RequireHealthOverlay/RequireHealthOverlay.js

```diff
@@ -8,26 +8,42 @@ import PropTypes from 'prop-types';
 
 import withHealth, { STATUS } from '../utils/withHealth';
 import loading from '../assets/img/icons/loading.svg';
+import { HealthModal } from './HealthModal';
 
 function statusMatches (status, require) {
+  const isSync = status === STATUS.GOOD;
+
+  const isNodeConnectedWithInternet =
+    status !== STATUS.DOWNLOADING &&
+    status !== STATUS.LAUNCHING &&
+    status !== STATUS.NO_NODE_CONNECTED_AND_NO_INTERNET &&
+    status !== STATUS.NODE_CONNECTED_AND_NO_INTERNET;
+
+  const isNodeConnectedNoInternet =
+    isSync ||
+    status === STATUS.NODE_CONNECTED_AND_NO_INTERNET ||
+    status === STATUS.NO_CLOCK_SYNC ||
+    status === STATUS.NO_PEERS ||
+    status === STATUS.SYNCING;
+
   switch (require) {
+    case 'connected-offline':
```

Comment:
I think the naming is confusing: do we require fether to be offline to show the underlying page?

I propose:
- `require='node'`: we need to be connected to the node
- `require='node-internet'`: we need to be connected to the node and have internet*
- `require='sync'`: we need to be sync.

\* we could also make the require prop an array: `require=['node', 'internet']`

Reply:
@amaurymartiny I've pushed a commit that uses this new naming. I haven't made the `require` prop an array though

Reply:
Cool, thanks. No problem for the array, it looks good as-is